src/System.Collections.Concurrent/tests/System.Collections.Concurrent.Tests.csproj

```diff
@@ -8,8 +8,8 @@
     <OutputType>Library</OutputType>
     <AssemblyName>System.Collections.Concurrent.Tests</AssemblyName>
     <RootNamespace>System.Collections.Concurrent.Tests</RootNamespace>
-    <DefineConstants Condition="'$(TargetGroup)'=='netstandard1.7'">$(DefineConstants);netstandard17</DefineConstants>
-    <NugetTargetMoniker Condition="'$(NugetTargetMoniker)'==''">.NETStandard,Version=v1.3</NugetTargetMoniker>
+    <DefineConstants Condition="'$(TargetGroup)'==''">$(DefineConstants);netstandard17</DefineConstants>
```

Comment:
Yeah, I guess that would be better but it is not super straight forward to make this with a script so it would have to be done manually. I can do that as a separate cleanup PR.